index.html

```diff
@@ -59,133 +67,197 @@ <h2 class="main-subtitle">I'am a Software Developer</h2>
 
 		<!-- Works Section -->
 		<section class="work-section">
-			<h2 class="work-title">My Recent Works</h2>
-			<div class="work-bar"></div>
+			<div class="work-text-container">
```

Comment:
- [ ] Please note the differences between your design and the Figma template shown below 🔽 as you need to add more spacing on the sides and reduce the space between the text and the line to match the Figma design, and you also need to pick the correct color.

### Figma Design
![Figma recent w](https://user-images.githubusercontent.com/104319462/220785572-a6804939-b190-4a68-8dee-ee3a396e269a.png)
### Your Design
![you recent w](https://user-images.githubusercontent.com/104319462/220785585-858f4533-d0ad-41f0-b5e0-ef82c70835d2.png)